packages/contracts/docs/osx/01-how-it-works/01-core/03-plugins/index.md

```diff
@@ -17,4 +17,37 @@ Plugins can be related to:
 
 - And **anything** else that comes to mind!
 
-In the next section, we will learn more about plugins, how they work, and who owns and manages them.
+## Understanding Plugins
+
+Whenever a DAO installs a plugin, an instance of that plugin's base template is deployed using the configuration parameters defined by the DAO. For example, you may want to use a specific token for your DAO's voting process, which means you have to determine this within your plugin's configuration parameters.
+
+Each instance of a plugin is installed to a DAO through the granting of permissions. They are usually managed by only one DAO.
+
+:::info
+Lern more about the different [plugin types](../../../02-how-to-guides/02-plugin-development/02-plugin-types.md) in our How-to guide.
+:::
+
+This raises questions on how the DAO manages plugins and who actually owns plugins..
+
+### How does the DAO Manage a Plugin?
+
+A DAO manages plugins and interactions between them, the DAO contract, and other parties with its. In more detail, its permission manager
+
+- controls the plugin setup process (installation, update, uninstallation)
```

Comment:
nice - changed "controls" to "enables" since the granting/revoking of these permissions is what enables the installation/uninstallation - even though it's controlled by the PSP.